exercise/birthday_paradox.py

```diff
@@ -0,0 +1,71 @@
+"""
+This module contains a code for exercises 10-8 related to:
+Think Python, 2nd Edition
+Chapter 10: Lists
+
+If there are 23 students in your class, what are the chances that two
+of you have thes same birthday? You can estimate this probability by 
+generating random samples of 23 birthdays and checking for matches.
+"""
+
+import random
+
+def has_duplicates(t):
+    """Returns True if any element appears more than once in a sequence.
+
+    t: list
+
+    returns: bool
+    """
+    # make a copy of t to avoid modifying the parameter
+    s = t[:]
+    s.sort()
+
+    # check for adjacent elements that are equal
+    for i in range(len(s)-1):
+        if s[i] == s[i+1]:
+            return True
+    
+    return False
+
+def random_bdays(n):
+    """Returns a list of integers between 1 and 365, with length n.
+
+    n: int
+
+    returns: list of int
+    """
+    t = []
```

Comment:
Function names, variable names must be meaningful.
Please double-check for all.